index.bs

```diff
@@ -3666,6 +3684,21 @@ credential.
         When storing a {{PublicKeyCredentialUserEntity/displayName}} member's value,
         the value MAY be truncated as described in [[#sctn-strings-truncation]]
         using a size limit greater than or equal to 64 bytes.
+
+    :   <dfn>language</dfn>
+    ::  An OPTIONAL [=resource-wide default=] language field [[!String-Meta]] for the entity.
+
+        The [=Client=] SHOULD verify that the value is a [=well-formed language tag=].
+
+        This member MAY be ignored if the combination of [=client=] and [=authenticator=] cannot suitably process and store it.
+
+    :   <dfn>direction</dfn>
+    ::  An OPTIONAL [=resource-wide default=] [=block direction=] field [[!String-Meta]] for the entity.
```

Comment:
The term here should be `string direction` instead of block direction.

```suggestion
    ::  An OPTIONAL [=resource-wide default=] [=string direction=] field [[!String-Meta]] for the entity.
```

Reply:
[String-Meta](https://w3c.github.io/string-meta/#resource_wide_default) specifies:

>If your specification defines its own document level defaults, provide two optional fields:
>
>- A resource-wide default language field SHOULD be called `language` [...]
>- A resource-wide default [block direction](https://w3c.github.io/string-meta/#dfn-block-direction) field SHOULD be called `direction` and support the values `ltr`, `rtl`, or `auto`.

So is "block direction" really not the appropriate term then?

Reply:
That might be a slight bug in our spec. The term isn't outright wrong, although I tend to think of "block direction" as the direction of text where the string is inserted and "string direction" as the base direction of the string (or, in this case, strings). I tend to think that "string direction" is clearer for implementers.

Reply:
Ok. Fixed in bc361f81940078bb0c023d90bc2f88c6cf91ab24, thanks!